public/src/news.js

```diff
@@ -0,0 +1,59 @@
+const app = document.getElementById('root')
+const newlist = document.querySelector('.new-list')
+
+const logo = document.createElement('img')
+logo.src='images/logo.png'
+
+const container = document.createElement('div')
+container.setAttribute('class', 'container')
+
+
+app.appendChild(logo)
+app.appendChild(container)
+
+
+fetch('https://content.guardianapis.com/search?show-blocks=all&api-key=3d1bd118-3d6f-4211-a61e-3d0f966b08bb')
+
+.then(response => {
```

Comment:
There should be no new line between these. You are calling the method `then` on the object returned by the function `fetch`.

Reply:
fixed.